### PR TITLE
Create the home and about pages

### PR DESCRIPTION
**Pull request by emmiiorji**
This change contains the website's home and about pages, styled with the [given template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as a guide. They're responsive to both mobile and desktop devices.

- No framework was used
- Each of these pages has versions for 2 different screen sizes:
    1. Mobile: up to 768px wide.
    2. Desktop: 768px or wider.

- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.
- The section "Featured speakers" is created dynamically in JavaScript.
- There is a JavaScript variable with the data about the speakers which is used when the page loads to create the HTML for the "Featured speakers" dynamically.

[Here's](https://www.loom.com/share/bc240d9bdb4f485f8ea18b83aee40216) a video I made introducing the project.

Also, the screenshot below shows the home page on a desktop device:
screenshot

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @emmiiorji , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing  linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      <a href="index.html" class="logo">PM Global <br />2022</a>
+      <div id="hamburger">
+        <div class="ham-bar"></div>
+        <div class="ham-bar"></div>
+        <div class="ham-bar"></div>
+      </div>
+      <div class="menu">
+        <div class="menu-element close">
+          <i class="fa fa-times fa-1x"></i>
+        </div>
```
Comment (codepantha) [inline index.html:38]: - [ ] The top dark bar is missing from your navigation design. Please reference the design template and make the appropriate changes 🙏 
### Your Design
image
### Behance Design
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      <img src="./assets/img/naver-Logo.png" alt="Naver Corporation">
+      <img src="./assets/img/daumkakao_icon.png" alt="Daumkakao">
+      <img src="./assets/img/airbnb-logo.png" alt="Air BNB">
+      <div class="partners-overlay"></div>
+    </div>
+  </section>
+  <footer>
+    <p class="copy">&copy;2022 Novustack.</p>
+    <p class="copyright">All rights reserved</p>
+  </footer>
```
Comment (codepantha) [inline index.html:116]: - [ ] The content of your footer is centered. Please align the logo to the left, and the text to the right as shown below 👇
### Your Design
image
### Behance Design
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          <img src="./assets/img/network-icon.png" alt="Network Icon">
+          <h2 class="card-title">PM Ignite</h2>
+          <p>Get opportunies to network with Novustack affiliate around the world, also after the summit</p>
+        </div>
+      </div>
+      <button type="button">Join PM Global Workshop 2022</button>
+    </section>
+
+    <section id="speakers"></section>
+
```
Comment (codepantha) [inline index.html:99]: - [ ] Each article in the speakers section should have only 2 columns (speaker cards) per row. In your case, you have 3 speakers per row as shown below: 👇 
### Your Design
image
### Behance Design
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  <main>
+
+  </main>
+  <section class="past-workshops">
+    <h2>See Past Workshops</h2>
+    
+    <hr class="heading-underline">
+    
+    <p>Take a look at the last two PM Global workshops that took place in Toronto, Canada and California</p>
+
```
Comment (codepantha) [inline about.html]: - [ ] The middle section with the header, divider, and logo missing from your design. Kindly see what you are missing below and implement it 👇 
### Behance Design
image

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  return `<div class="speaker-card">
+            <div class="grid-bg"></div>
+            <img src="${speaker.img}" class="speaker-img" alt="${speaker.name}">
+            <div class="speaker-info">
+              <h3 class="speaker-name">${speaker.name}</h3>
+              <p class="speaker-title">${speaker.title}</p>
+              <hr class="sep-info">
+              <p class="speaker-bio">${speaker.bio}</p>
+            </div>
+          </div>`;
```
Comment (codepantha) [inline js/script.js]: - [ ] Please make your speaker card an `<article>` tag as it is much more semantically meaningful in this case than a `<div>`

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @emmiiorji 👋, 

## STATUS: REQUESTED CHANGES ♻️

Excellent job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🎯 💡 
- Beautiful UI ✔️ 
- Meaningul commit messages ✔️ 
- Professional README ✔️ 
- Linter checks all green ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please record a maximum of 5 min long video detailing the features of your project.
- [ ] Please give your PR a detailed title. The current title 'Development' is not good enough as it doesn't give any hint as to what changes the PR might introduce.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (danielufeli) [reply index.html:99]: [ ] - Kindly make these section 2 columns as requested by the previous reviewer

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @emmiiorji,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️
- Professional Readme ✔️

## Required Changes ♻️
 - [ ] Please record a maximum of 5 min long video detailing the features of your project.

_Check the comments under the review._

 Alt text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me with @danielufeli. in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emmiiorji** [comment]
Hi @danielufeli 

Thanks for your kind review.

I had already updated the pull request description with the [link to my loom video](https://www.loom.com/share/bc240d9bdb4f485f8ea18b83aee40216) before making another review request.

For the [number of columns](https://github.com/emmiiorji/project-mgt-workshop/pull/4#discussion_r857030732), here's what my browser displays:

image

Kindly check these from your end again and let me know what you have.
Thank you.

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @emmiiorji , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**